Newsletter/style.css

```diff
@@ -94,7 +94,7 @@ h1 {
     color:white;
     text-align:center;
     margin-bottom:5%;
-    transform:translateY(-2300%);
+    transform:translateY(-30em);
   }
```

Comment:
whats the need of this change ?

Reply:
because of this , even though it was not happening in my side


![Opera Snapshot_2022-04-05_161514_github com](https://user-images.githubusercontent.com/82874704/161737606-26ecb787-86c5-4b62-888b-3fde1c347554.png)